notebooks/simple-feature-transformations-explain-local.ipynb

```diff
@@ -0,0 +1,512 @@
+    "    1. Train a binary classification model\n",
+    "    1. Explain the model on raw features\n",
+    "        1. Generate global explanations\n",
+    "        1. Generate local explanations\n",
+    "1. [Visualize results](#Visualize)\n",
+    "1. [Next steps](#Next%20steps)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='Introduction'></a>\n",
+    "## 1. Introduction\n",
+    "\n",
+    "This notebook illustrates how to use interpret-community to help interpret a binary classification model using an IBM dataset with employee attrition data. It uses one-to-one and one-to-many feature transformations from the raw data to engineered features. The one-to-many feature transformations include one hot encoding for categorical features. The one-to-one feature transformations apply standard scaling on numeric features. The interpret-community tabular data explainer in this repo is then used to obtain raw feature importances.\n",
+    "\n",
+    "Three tabular data explainers are demonstrated: \n",
+    "- TabularExplainer (SHAP)\n",
+    "- MimicExplainer (global surrogate)\n",
+    "- PFIExplainer\n",
+    "\n",
+    "| ![Interpretability Toolkit Architecture](./img/interpretability-architecture.png) |\n",
+    "|:--:|\n",
+    "| *Interpretability Toolkit Architecture* |\n",
+    "\n",
+    "<a id='Project'></a>       \n",
+    "## 2. Project\n",
+    "\n",
+    "The goal of this project is to classify employee attrition with scikit-learn and locally running the model explainer: \n",
+    "\n",
+    "1. Transform raw features to engineered features\n",
+    "2. Train a SVC classification model using Scikit-learn\n",
+    "3. Run 'explain_model' globally and locally with full dataset in local mode, which doesn't contact any Azure services.\n",
+    "4. Visualize the global and local explanations with the visualization dashboard.\n",
+    "\n",
+    "<a id='Setup'></a>\n",
+    "## 3. Setup\n",
+    "\n",
+    "If you are using Jupyter notebooks, the extensions should be installed automatically with the package.\n",
+    "If you are using Jupyter Labs run the following command:\n",
+    "```\n",
+    "(myenv) $ jupyter labextension install @jupyter-widgets/jupyterlab-manager\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='Explain'></a>\n",
+    "## 4. Run model explainer locally at training time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.pipeline import Pipeline\n",
+    "from sklearn.impute import SimpleImputer\n",
+    "from sklearn.preprocessing import StandardScaler, OneHotEncoder\n",
+    "from sklearn.svm import SVC\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "\n",
+    "# Explainers:\n",
+    "# 1. SHAP Tabular Explainer\n",
+    "from interpret.ext.blackbox import TabularExplainer\n",
```

Comment:
I guess we'll just try to update the use of `interpret.ext` whenever we know that change can be made.

Reply:
yes, we'll keep it until it is updated.